packs/st2cd/actions/git_branch.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+GIT=`which git`
+REPO=$1
+DATE=`date +%s`
+VERSION=$2
+BRANCH=$3
+FILE="st2common/st2common/__init__.py"
+OUTPUT=/tmp/git-output-$DATE
+
+cd ${REPO}
+OUT=`$GIT pull origin master -q > $OUTPUT && $GIT branch ${BRANCH} -q && $GIT checkout ${BRANCH} -q`
+if [[ $? == 0 ]]
+then
+  VERSIONOUT=`sed -i '' "s/\(__version__ = \).*/\1'${VERSION}'/" ${FILE}`
```

Comment:
@DoriftoShoes HALP. This doesn't work correctly and the action fails. Can you help fix this line?